strftime.js

```diff
@@ -595,10 +595,8 @@
             else
                 weekday--;
         }
-
-        var firstDayOfYearUtc = Date.UTC(date.getFullYear(), 0, 1),
-            dateUtc = Date.UTC(date.getFullYear(), date.getMonth(), date.getDate()),
-            yday = Math.floor((dateUtc - firstDayOfYearUtc) / 86400000),
+        var firstDayOfYear = new Date(date.getFullYear(), 0, 1),
```

Comment:
The intention was to compare date without taking DST into account. `date.getDate()` returns proper day with DST. But the difference calculated without DST. 

Could you point when the bug could appear here please. I'm not sure I get you.